package.json

```diff
@@ -42,7 +42,8 @@
   },
   "scripts": {
     "bench": "matcha",
-    "test": "standard && tape test/*.test.js | tap-spec"
+    "test": "standard && tape test/*.test.js | tap-spec",
+    "test:new": "tape null-values.test.js | tap-spec"
```

Comment:
I had to add a "test/" in front of "null-values.test.js" here because tape wasn't able to find the test case. Can you reproduce that?